lib/parser/index.js

```diff
@@ -0,0 +1,79 @@
+const sketch2json = require('sketch2json');
+const { getClient, figma2jsonFromAPI, findIds } = require('../figma');
+const {
+  getPageArrays,
+  getColorsFromArtboard,
+  getGradientsFromArtboard,
+  getShadowsFromArtboard,
+  getBordersFromArtboard,
+} = require('../../lib/sketch');
+const { readFile, uniqueArray } = require('../utils');
+
+const parseTokens = async (args, flags) => {
+  const [file] = args;
+  const isSketch = /.*sketch/.test(file);
```

Comment:
I think this regex is supposed to be:

```suggestion
  const isSketch = /*.sketch/.test(file);
```

Reply:
Don't you mean 
```js
/.*\.sketch/
```